src/Compatibility/Core/src/iOS/Renderers/ImageAnimationHelper.cs

```diff
@@ -52,9 +53,9 @@ public void AddFrameData(int index, CGImageSource imageSource)
 				using (var delayTimeValue = gifImageProperties?.ValueForKey(ImageIO.CGImageProperties.GIFDelayTime))
 				{
 					if (unclampedDelayTimeValue != null)
-						double.TryParse(unclampedDelayTimeValue.ToString(), out delayTime);
+						double.TryParse(unclampedDelayTimeValue.ToString(), NumberStyles.Any, CultureInfo.InvariantCulture, out delayTime);
```

Comment:
Fix issue with gifs and differents DefaultThreadCurrentCulture on iOS
https://github.com/xamarin/Xamarin.Forms/pull/14698